albumentations/core/composition.py

```diff
@@ -172,6 +172,8 @@ def __init__(
         self.is_check_args = True
         self._disable_check_args_for_transforms(self.transforms)
 
+        self.is_check_shapes = True
```

Comment:
Please add this argument to `_to_dict` and to `__init__` args this will be useful for users.

Reply:
Would be great if you add this parameter to serialization test. But it is not necessary.

Reply:
I think we don't need a separate test for this parameter since tests/test_serialization.py __test_serialization_v2_to_dict__ will cover it. What do you think?